### PR TITLE
Feat: Project Euler Problem 7

### PR DESCRIPTION
**Pull request by domix80**
Add resolution for Euler Problem 007

### CODE REVIEW

**Comment by domix80** [comment]
@aparibocci, Could you please review it?

**Comment by domix80** [comment]
@StepfenShawn - May I ask you to review and eventually merge the code?

File: project_euler/problem_007/sol1.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#Project Euler Problem 7: #https://projecteuler.net/problem=7
```
Comment (StepfenShawn) [inline project_euler/problem_007/sol1.rb:1]:
```suggestion
#Project Euler Problem 7:  https://projecteuler.net/problem=7
```

File: project_euler/problem_007/sol1.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+#10001st prime
+#By listing the first six prime numbers: 2, 3, 5, 7, 11, #and 13, we
```
Comment (StepfenShawn) [inline project_euler/problem_007/sol1.rb:3]:
```suggestion
#By listing the first six prime numbers: 2, 3, 5, 7, 11, and 13, we
```

**Review by StepfenShawn — CHANGES_REQUESTED** [review]

**Review by StepfenShawn — APPROVED** [review]
thanks!

**Pull request merged**